ruby/vampires.rb

```diff
@@ -0,0 +1,77 @@
+attempts = 0
+
+until attempts == 4
+	attempts += 1
+
+puts "What is your name?"
+name = gets.chomp
+	
+
+def age_finder(x, y)
+		 (2016 - y.to_i) == x.to_i
+end
+
+puts "How old are you? What year were you born?"
+age = gets.chomp
+year = gets.chomp
+
+	
+puts "The cafeteria serves garlic bread. Should we order some for you?"
+garlic_bread = gets.chomp
+
+	if garlic_bread == "yes"
```

Comment:
There is some refactoring that could be done with setting the variables.   For instance: 
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
    garlic_bread = true
else
   garlic_bread = false
end
```
can be refactored to:
```
garlic_bread = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `garlic_bread` will be set to false.